ctools/defragment_sharded_collection.py

```diff
@@ -360,10 +360,13 @@ def lookahead(iterable):
             progress.update()
 
             if len(consecutive_chunks) == 0:
-                consecutive_chunks = [c]
-                estimated_size_of_consecutive_chunks = get_chunk_size(c)
 
-                if not has_more:
+                # Assume that the user might run phase I more than once. We may encouter chunks with 
+                # defrag_collection_est_size set and minimum 75% target chunk size. Do not attempt 
+                # to merge these further
+                skip_chunk = get_chunk_size(c) >= target_chunk_size_kb * 0.75
```

Comment:
If we run the script for the first time , `get_chunk_size()` will always return the estimated chunk size (aka `args.phase_1_estimated_chunk_size_kb`). So if the user didn't set it it will always be `25.6MiB`. I guess here we want to use `coll.data_size_kb_from_shard` when the chunk didn't already have the `defrag_collection_est_size` field.

Reply:
Before `args.phase_1_estimated_chunk_size_kb` was only used to optimistically decide when to actually check the size of the range in consecutive_chunks array. With this changes we are relying on the `args.phase_1_estimated_chunk_size_kb` in order to definitely skip a chunk without calculating its actual size ever.

Reply:
if the target chunk size is only `1/0.75 =  1.3` bigger than `args.phase_1_estimated_chunk_size_kb` it would not make sense to use the script I think.
I suppose we could change it to only do a skip if the size was actually estimated

Reply:
The targetChunkSize in my test case is `1MiB` while `args.phase_1_estimated_chunk_size_kb` is `25.6MiB`